scripts/generate-wallet-list.ts

```diff
@@ -77,6 +77,15 @@ export const iosList: App[] = [
     logo: 'ios-airgap.png',
     universalLink: 'https://wallet.airgap.it',
     deepLink: 'airgap-wallet://'
+  },
+  {
+    key: 'naan_ios',
+    name: 'Naan - A Tasty Tezos Wallet',
```

Comment:
Please use only the wallet name and remove the additional tag line.

Reply:
I have updated name to "Naan wallet".